tweets-cleaner.js

```diff
@@ -2,10 +2,15 @@
 
 const chalk = require('chalk')
 const Twitter = require('twitter')
-const Converter = require('csvtojson').Converter
 const jsonfile = require('jsonfile')
 const config = require('./config')
 
+function getTweets () {
+  global.window = { YTD: { tweet: { } } }
+  const tweets = require(config.path)
+  return window.tweets.YTD.tweet.part0
```

Comment:
Thanks for this PR!

Seems it should be 👇 , as @tiaratsalsaa suggests:

```suggestion
  return window.YTD.tweet.part0
```